setup.py

```diff
@@ -126,6 +126,9 @@
         'invenio_base.apps': [
             'invenio_oauthclient = invenio_oauthclient:InvenioOAuthClient',
         ],
+        'invenio_base.api_apps': [
```

Comment:
What do you mean exactly? The sender should be the `api_app`?
That's what I did: 
 1. I registered through the `app` 
 2. I made a REST call to `api_app`. And here is failing.